snippets/csharp/programming-guide/strings/string_program_1.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace StringCompare
+{
+    class Program
+    {
+        static void Main()
+        {
+            string jupiterFirst = "Jupiter";
+            string jupiterLast = "Hammon";
+            int jupiterBorn = 1711;
+            int jupiterPub = 1761;
+            // Variables
+            Console.WriteLine($"{jupiterFirst} {jupiterLast} was an African American poet born in {jupiterBorn}.");
+            // Simple expression
+            Console.WriteLine($"He was first published in {jupiterPub} at the age of {jupiterPub - jupiterBorn}.");
+            // Complex Expressions
```

Comment:
nit: if it's possible to avoid comments in code, don't include them. Comments in the code are not translated when the docs are localized into other languages. I think, in this example comments can be dropped. Of course, comments with the output is an exception.

Reply:
Removed except for output.